pkg/cluster/check/default.go

```diff
@@ -29,6 +29,13 @@ func DefaultClusterChecks() []ClusterCheck {
 			}, 5*time.Minute, 5*time.Second)
 		},
 
+		// wait for etcd members to be the control plane nodes
+		func(cluster ClusterInfo) conditions.Condition {
+			return conditions.PollingCondition("etcd members to be control plane nodes", func(ctx context.Context) error {
```

Comment:
`checking for etcd members to be control plane nodes` i think makes more sense?

Reply:
This part of the string is prepended by `waiting for ` by another place so that the final string renders as `waiting for etcd members to be control plane nodes`. See the other checks, they all use the same form.

Reply:
Ah you mean removing the `the`?

Reply:
ahh, that makes sense then, just wanted to confirm